app/src/main/res/layout/activity_progress.xml

```diff
@@ -57,4 +57,6 @@
         style="@style/start_cooking_sticky_button"
         />
 
+    
+
 </RelativeLayout>
```

Comment:
For some reason, GitHub doesn't like not having one empty line at the end. Can you add one please?